src/CFamily/Rules/ICFamilyRulesConfig.cs

```diff
@@ -50,6 +51,15 @@ public class RuleMetadata
 
         [JsonProperty("compatibleLanguages")]
         public string[] CompatibleLanguages { get; set; }
+
+        [JsonProperty("code")]
+        public Code Code { get; set; }
+    }
+
+    public class Code
+    {
+        [JsonProperty("impacts")]
+        public Dictionary<SoftwareQuality, SoftwareQualitySeverity> Impacts { get; set; } = new Dictionary<SoftwareQuality, SoftwareQualitySeverity>();
```

Comment:
That would be just testing the Newtonsoft there weren't any tests for old severity either.

Reply:
I think for that purpose an integration test like we do in the rules description makes sense.  But I would argue it would be Out of Scope for this sprint but we can tackle it when we are going to refactor the pipelines.